google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -120,6 +121,15 @@ public CancellableRunnable createDeployRunner(
       throw new RuntimeException("Invalid deployment source selected for deployment");
     }
 
+    if (CloudSdkService.getInstance().validateCloudSdk().contains(
```

Comment:
@aslo @patflynn @etanshaul because AppEngineDeployStandardTask only handled the standard case, I moved the check here so it also works for Flex.